contributors/design-proposals/propagation.md

```diff
@@ -109,6 +125,12 @@ Opinion against this:
 
 1. (From @jonboulle) May cause cross-runtime compatibility issue.
 
+1. It's not possible to validate a pod + mount propagation. Mount propagation
```

Comment:
But even then, it will not work out of the box for hostpath PVs.  There are limitations in how the hostpath volume plugin is implemented that prevent it from working in a multi-node cluster.